configurations/linux-socket-ip-driver.asn

```diff
@@ -12,7 +12,9 @@ LINUX-SOCKET-IP-DRIVER DEFINITIONS AUTOMATIC TAGS ::= BEGIN
 
 Port-T ::= INTEGER (0 .. 65535)
 
-Version-T  ::= ENUMERATED {ipv4, ipv6}
+Version-T ::= ENUMERATED {ipv4, ipv6}
+
+Packetizer-T ::= ENUMERATED {default, ccsds, thin}
```

Comment:
Why is this needed?

Reply:
When I tried to use ASN simple type (one of the strings), I got an error message, that I have to use 'named' type.

Reply:
I still don't think this is needed. Let's talk.

Reply:
Done. The code is simplified now, and packetizers are stored in kazoo only.

https://gitrepos.estec.esa.int/taste/kazoo/-/merge_requests/99